### ISSUE TITLE
Change or Remove UID assigned in Dockerfile 

### ISSUE THREAD

**Issue by amaurer** [openethereum/openethereum#9760]
- **Parity Ethereum version**: 2.0.6
- **Operating system**: Centos7
- **Installation**: Docker parity/parity:stable
- **Fully synchronized**: yes
- **Network**: mainnet
- **Restarted**: yes

**actual** 
Parity runs as a user on host that already is assigned UID 1000. This causes issues when volume mounting a host directory.

**expected behavior** 
Parity runs as an unknown user id.  

Seems like UID 1000 and GID 1000 is not a great choice as it is likely there is already a user assigned those IDs.  Is there a great choice in ID to pick? I'm not sure, maybe leave that up to the people using the software. 

**steps to reproduce**
1. create a user with UID 1000
2. start in Docker
3. view parity process running as user already assigned uid 1000

**Comment by ddorgan** [comment]
@amaurer just to check if this works, can you create the directory on the host system before running the docker container and see if this resolves the issue. Thanks...

**Comment by amaurer** [comment]
Yes, I do that.
The issue is that uid is already assigned to another user which has sudo access. So it was odd, when upgrading Parity, to see it running, as an already specified user, on the host. 

This issue really is just an annoyance, thanks for marking it as such.  But I had to spend time chasing this down as I thought it was a bigger issue. 

I resolved by creating a user "ethereum" as uid 1200 on the host with the proper permissions to the directory.  In my Dockerfile. I also changed Parity user to uid 1200 (instead of it's set 1000 uid) in said Dockerfile.
